src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.main__calendar {
+  margin: auto;
+  padding-top: 150px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  width: $widthCalendar;
+  gap: $gapSize;
+
+  @each $day, $marg in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $marg;
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index} :nth-child(n + #{$index + 1}) {
+      display: none;
+    }
+  }
+    
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square($cellSize);
+    background-color: #eee;
+    border: 1px solid #000;
+    cursor: pointer;
+    transition: background-color 3s, transform 2s;

```

3s & 2s - it's too long
let's put 0.3s & create variable for transition duration
```suggestion
    transition: background-color $transitionDuration, transform $transitionDuration;

```

---

Changed. Thank's